pkg/cloudclient/aws/private.go

```diff
@@ -266,7 +266,7 @@ func (c Client) findUnreachableEndpoints(ctx context.Context, instanceID string)
 		Latest:     &latest,
 	}
 
-	err := helpers.PollImmediate(30*time.Second, 10*time.Minute, func() (bool, error) {
+	_ = helpers.PollImmediate(30*time.Second, 10*time.Minute, func() (bool, error) {
```

Comment:
We're not catching the err here any more because we're relying on the `output` struct right? Can you leave a comment explaining this?

Reply:
yup, actually I can take note at appropriate places. I think there are more than one where we fill the output instead of returning err